docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+    "\n",
+    "### Defining the research question\n",
+    "\n",
+    "Our research question depends entirely on the context.\n",
+    "We want to answer a question that is relevant to\n",
+    "the stakeholders and decision-makers we are advising,\n",
+    "so we determine the question through dialogue with them.\n",
+    "Often, this starts from a proposal on their side.\n",
+    "The client may come to us with a question like this:\n",
+    "\n",
+    "> How much impact could it have if we scaled up SQ-LNS in Ethiopia?\n",
+    "\n",
+    "However, this question is **not** precise enough to be our research question.\n",
+    "We need to ask clarifying questions to understand what options decision-makers\n",
+    "really have and how a simulation result could help them choose.\n",
+    "For example, in a perfect world we might want 100% of children to receive SQ-LNS,\n",
+    "but maybe this isn't a feasible option due to funding constraints\n",
+    "and practical challenges in distribution.\n",
+    "\n",
+    "We might ask:\n",
+    "\n",
+    "- How much SQ-LNS scale-up could be funded?\n",
+    "- How long would the scale-up take?\n",
+    "- Would it be targeted to a specific population or given to everyone?\n",
+    "- What do we mean by \"impact\"? What units would this be measured in?\n",
+    "- Are we interested in the impact over 5 years? 10 years? The entire lifetimes of the kids who get the intervention?\n",
+    "\n",
+    "Of course, in this tutorial the scenario is invented and we don't have a client\n",
+    "to ask these questions.\n",
+    "However, imagine that we did, and after conversations with them we land on the following research question:\n",
+    "\n",
+    "> How many [disability-adjusted life years](https://en.wikipedia.org/wiki/Disability-adjusted_life_year) (DALYs)\n",
+    "> between 2025 and 2030 would be averted by\n",
+    "> delivering SQ-LNS to all eligible children in Ethiopia starting in 2025?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Making a causal diagram\n",
+    "\n",
+    "Answering this question seems quite difficult --\n",
+    "after all, there are no data from our years of interest (since they are in the future),\n",
+    "and in order to isolate the **effect** of the scale-up, we'd want to\n",
+    "compare a universe where SQ-LNS was delivered with one where it wasn't.\n",
+    "\n",
+    "The only way to get out of this conundrum is to apply our understanding of **how** our intervention\n",
+    "could have an impact.\n",
+    "Some of this understanding was required to form the research question in the first\n",
+    "place; we wouldn't be asking the question unless we knew of a **mechanism** by which\n",
+    "the action (delivering SQ-LNS) could cause the outcome (averted DALYs) in\n",
+    "the context (Ethiopia, 2025-2030).\n",
+    "\n",
+    "But let's start from scratch. From the [SQ-LNS FAQs](https://sqlns.ucdavis.edu/FAQs):\n",
```

Comment:
maybe "From the UC Davis SQ-LNS FAQs". (unless that link is somehow considered _the_ SQ-LNS FAQs?)

Reply:
@SylLutze correct me if I'm wrong but I do think that is _the_ SQ-LNS website (published by the SQ-LNS Task Force)

Reply:
Hmm good question. It is run by UC Davis. It is the main task force though and leads a lot of the work for SQ-LNS globally although there are plenty of other folks that do work in this area too. But I think most people would think of this as the leader.